src/java.base/share/classes/java/lang/Math.java

```diff
@@ -819,8 +819,8 @@ public static int round(float a) {
      * equal to the value of {@code Long.MAX_VALUE}.</ul>
      *
      * @apiNote
-     * This method corresponds to the convertToIntegerTiesToEven
-     * operation defined in IEEE 754.
+     * This method corresponds to the convertTowardPositive operation
+     * defined in IEEE 754.
```

Comment:
Probably what is meant is IEEE `convertToIntegerTowardPositive` (not `convertTowardPositive`).
However, that's another kind of rounding, which always rounds any non-integral value toward positive infinity, not just ties. This is not what happens with this method.

Reply:
> not just ties
Plz. disregard this 3 words

Reply:
Okay, checking the IEEE 754-2019 spec again, there are operations to round a floating-point value to an integral-valued floating-point value (roundToIntegral{$ROUNDING_DIRECTION_ATTRIBUTE} and separately operations to round from a floating-point value to a value in an integer format (convertToInteger{$ROUNDING_DIRECTION_ATTRIBUTE} ). However, as noted, there is no IEEE 754 operations for rounding up only for ties. I'll remove the API notes for the round methods; thanks for catching this.